shapeshift/web
# fix: fox and foxy balances on fox page respect multiple accounts

## Description



* we were using a selector that didn't take accountId as an input, multi account balances were wrong
* updates to `cryptoHuman` vernacular
* `MainOpportunity` component is still hardcoded to account 0

## Notice




## Pull Request Type


## Issue (if applicable)



n/a

## Risk

* isolated to the fox page
* we show incorrect fox/foxy balances on the fox page



## Testing

* have multi account flag off
* visit fox page with an account with fox and/or foxy balance
* see the balances are correct
* turn multi account flag on
* have a second eth account with fox and/or foxy balance
* visit fox page and see fox and foxy page total balances at the top show correctly as the sum of
  both accounts


### Engineering

* see common steps



### Operations

* see common steps



## Screenshots (if applicable)
